.ci/pipelines/e2e-tests-snapshot-versions-gke.Jenkinsfile

```diff
@@ -58,7 +58,7 @@ pipeline {
                 )}"""
             }
             parallel {
-                stage("7.16.0-SNAPSHOT") {
+                stage("7.17.0-SNAPSHOT") {
```

Comment:
https://github.com/elastic/cloud-on-k8s/blob/df70d860ccef9463da7005adf078401328cc117f/.ci/pipelines/e2e-tests-snapshot-versions-gke.Jenkinsfile#L68

must also be modified.

Reply:
😞 of course